cli/flox/doc/flox-build.md

```diff
@@ -0,0 +1,179 @@
+---
+title: FLOX-BUILD
+section: 1
+header: "Flox User Manuals"
+...
+
+```{.include}
+./include/experimental-warning.md
+```
+> Feature flag: `build`
+q
+# NAME
+
+flox-build - build packages
+
+# SYNOPSIS
+
+```
+flox [<general-options>] build
+     [-d=<path>]
+     [-L]
+     [<package>]...
+```
+
+# DESCRIPTION
+
+Build the specified `<package`> to run or verify your software locally.
+Create build artifacts from instructions in the manifest
+of the environment in `<path>`.
+
+## Manifest defined Packages
+
+Possible values for `<package>` are all keys under the `build` attribute
+in the `manifest.toml`.
+If no `<package>` is specified, flox will attempt to build all packages
+that are defined in the environment.
+
+Packages are built by running the script defined in `build.<package>.command`
+within a `bash` subshell.
+The shell will behave as if `flox activate` was run
+immidiately prior to running the build script.
+The current working directory of the script will be set to the root
+of the `git` repository in which the environment is defined.
+Consequently, _building requires your environment to be tracked
+under `git` version control_.
```

Comment:
nonblocking: explain why (I don't even know)

Reply:
fwiw i dont think the sentence before that is even truthful currently, ive been buildin in a subdir and _not_ have the cwd set to anything but the dir containing the .flox (which tbh, is the more sensible behavior in my opinion)
The restriction also only concerns sandboxed builds. i'll remove this here and amend something in the paragraph below.